htmd/molecule/molecule.py

```diff
@@ -1955,9 +1955,17 @@ def __init__(self, sel=None, style=None, color=None):
 
 
 class TestMolecule(TestCase):
-    def test_trajReadingAppending(self):
+    def setUp(self):
+        super().setUp()
         from htmd.home import home
 
+        self.trajmol = Molecule(path.join(home(dataDir='metricdistance'), 'filtered.pdb'))
+        self.trajmol.read(path.join(home(dataDir='metricdistance'), 'traj.xtc'))
+
+        self.mol3PTB = Molecule('3PTB')
```

Comment:
As a note for the future, we should be using `subTest` when working on the same object, to avoid loading the Molecule over and over again. It's not needed now.